code/src/OSFramework/Interface/IDataSource.ts

```diff
@@ -83,5 +85,7 @@ namespace OSFramework.Grid {
          * @param value
          */
         trimSecondsFromDate(value: string): string;
+
+        updateAddedRowKey(currentRowId: string, newKey: string): boolean;
```

Comment:
Can you please add a comment here so that we have a little help while using the method in the future?